md/forms.md

```diff
@@ -24,3 +24,18 @@ The overview consists of three main sections:
 * Timeline: it shows in chronological order information about all the actions that have been performed in the selected case.
 
 You can see how to manage the link between process and and form in the documenation about [live update](live-update.md)
+
+### Application Theme access
+
+If your forms is viewed in an application, you will have access facilities for [the application theme](applications.md).
+
+The `Theme.css` is directly accessible by adding the following link in `index.html`: `<link href="../theme/theme.css" rel="stylesheet" />`
+This link is present by default in the forms generated by the UI Designer. 
+
+::: info 
+Theme is loaded via the `app` URL parameter. The living application layout inject this `app` URL parameter in the targeted page/form URL.
```

Comment:
The living application layout injects this `app` URL parameter in the targeted page/form URL.
("s" added to "inject")